assets/charts/components/headlamp/templates/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "headlamp.fullname" . }}
+  labels:
+    {{- include "headlamp.labels" . | nindent 4 }}
+  namespace: kube-system
```

Comment:
Namespace should not be set unless you want this resource to be explicitly created in the namespace other than release namespace.